app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1321,25 +1330,31 @@ class BrowserTabViewModel @Inject constructor(
     }
 
     private suspend fun updateLoadingStatePrivacy(domain: String) {
-        val isAllowListed = isAllowListed(domain)
+        val privacyProtectionEnabled = isPrivacyProtectionDisabled(domain)
```

Comment:
This line is confusing. Maybe rename `privacyProtectionEnabled` to `privacyProtectionDisabled`.